### ISSUE TITLE
alt text should describe the image

### ISSUE THREAD

**Issue by discospiff** [pkkenn/IS7012#2]
I see you have alt text... go just a bit further on this and describe what the image looks like for someone who cannot see it.

https://github.com/pkkenn/IS7012/blob/ac95cdaa7366ecfa9f22e3045532ad4170e8ecbe/Week 2/RecruitCatAlexiskh/Pages/Index.cshtml#L45

**Comment by pkkenn** [comment]
[https://github.com/pkkenn/IS7012/blob/fdd2b2b3d064543c31d9fe5e5310ab9e9ed24b89/Week%203/RecruitCatAlexiskh/Pages/Index.cshtml#L45](https://github.com/pkkenn/IS7012/blob/fdd2b2b3d064543c31d9fe5e5310ab9e9ed24b89/Week%203/RecruitCatAlexiskh/Pages/Index.cshtml#L45)
